Missing `quarkus.oidc.tls.key-store-provider` and `quarkus.oidc.tls.trust-store-provider` properties on OIDC extension

### Describe the bug

I would like to use OIDC with mtls over FIPS. 

The certificates that I am thinking to use are `BCFIPSJSSE` (BouncyCastleJsseProvider) as we do in other [modules](https://quarkus.io/guides/security-customization#bouncy-castle-jsse-fips). The problem that I have is that there is no `quarkus.oidc.tls.key-store-provider` / `quarkus.oidc.tls.trust-store-provider`. 

application.properties

```
quarkus.http.auth.permission.authenticated.paths=/*
quarkus.http.auth.permission.authenticated.policy=authenticated
quarkus.oidc.token.lifespan-grace=5
quarkus.oidc.token.principal-claim=email
quarkus.oidc.token.issuer=${quarkus.oidc.auth-server-url}
quarkus.oidc.tls.verification=certificate-validation
quarkus.oidc.tls.key-store-file=client-bcfips-keystore.jks
#quarkus.oidc.tls.key-store-file=client-keystore.jks
quarkus.oidc.tls.key-store-file-type=BCFKS
quarkus.oidc.tls.key-store-password=password

quarkus.oidc.tls.trust-store-file=client-bcfips-truststore.jks
#quarkus.oidc.tls.trust-store-file=client-truststore.jks
quarkus.oidc.tls.trust-store-file-type=BCFKS
quarkus.oidc.tls.trust-store-password=password
```
OIDC configuration doc ref: https://quarkus.io/guides/all-config#quarkus-oidc_quarkus-oidc-openid-connect

The issue that I am trying to avoid is something like: 
```
Caused by: java.security.KeyStoreException: BCFKS not found
```

--- a/extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java
+++ b/extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java
@@ -440,6 +440,14 @@ public class OidcCommonConfig {
         public Optional<String> keyStoreFileType = Optional.empty();
 
         /**
+         * An optional parameter to specify a provider of the key store file. If not given, the provider is automatically
+         * detected
+         * based on the key store file type.
+         */
+        @ConfigItem
+        public Optional<String> keyStoreProvider;
+
+        /**
          * A parameter to specify the password of the key store file. If not given, the default ("password") is used.
          */
         @ConfigItem(defaultValue = "password")
@@ -484,6 +492,14 @@ public class OidcCommonConfig {
         @ConfigItem
         public Optional<String> trustStoreFileType = Optional.empty();
 
+        /**
+         * An optional parameter to specify a provider of the trust store file. If not given, the provider is automatically
+         * detected
+         * based on the trust store file type.
+         */
+        @ConfigItem
+        public Optional<String> trustStoreProvider;
+
         public Optional<Verification> getVerification() {
             return verification;
         }
@@ -514,6 +530,22 @@ public class OidcCommonConfig {
 
         public void setTrustStoreCertAlias(String trustStoreCertAlias) {
             this.trustStoreCertAlias = Optional.of(trustStoreCertAlias);
+        }
+
+        public Optional<String> getKeyStoreProvider() {
+            return keyStoreProvider;
+        }
+
+        public void setKeyStoreProvider(String keyStoreProvider) {
+            this.keyStoreProvider = Optional.of(keyStoreProvider);
+        }
+
+        public Optional<String> getTrustStoreProvider() {
+            return trustStoreProvider;
+        }
+
+        public void setTrustStoreProvider(String trustStoreProvider) {
+            this.trustStoreProvider = Optional.of(trustStoreProvider);
         }
 
     }

--- a/extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonUtils.java
+++ b/extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonUtils.java
@@ -131,7 +131,8 @@ public class OidcCommonUtils {
                         .setPassword(oidcConfig.tls.getTrustStorePassword().orElse("password"))
                         .setAlias(oidcConfig.tls.getTrustStoreCertAlias().orElse(null))
                         .setValue(io.vertx.core.buffer.Buffer.buffer(trustStoreData))
-                        .setType(getStoreType(oidcConfig.tls.trustStoreFileType, oidcConfig.tls.trustStoreFile.get()));
+                        .setType(getStoreType(oidcConfig.tls.trustStoreFileType, oidcConfig.tls.trustStoreFile.get()))
+                        .setProvider(oidcConfig.tls.trustStoreProvider.orElse(null));
                 options.setTrustOptions(trustStoreOptions);
                 if (Verification.CERTIFICATE_VALIDATION == oidcConfig.tls.verification.orElse(Verification.REQUIRED)) {
                     options.setVerifyHost(false);
@@ -150,7 +151,8 @@ public class OidcCommonUtils {
                         .setAlias(oidcConfig.tls.keyStoreKeyAlias.orElse(null))
                         .setAliasPassword(oidcConfig.tls.keyStoreKeyPassword.orElse(null))
                         .setValue(io.vertx.core.buffer.Buffer.buffer(keyStoreData))
-                        .setType(getStoreType(oidcConfig.tls.keyStoreFileType, oidcConfig.tls.keyStoreFile.get()));
+                        .setType(getStoreType(oidcConfig.tls.keyStoreFileType, oidcConfig.tls.keyStoreFile.get()))
+                        .setProvider(oidcConfig.tls.keyStoreProvider.orElse(null));
                 options.setKeyCertOptions(keyStoreOptions);
 
             } catch (IOException ex) {

